### PR TITLE
Add Inception resnet v2 node

### LINKED ISSUE #19: Add node for Inception ResNet v2
https://developer.ibm.com/exchanges/models/all/max-inception-resnet-v2/

### PR DESCRIPTION
**Pull request by ptitzler**
Closes #19.   This PR delivers
 - the customized source code
 - a test script (covering all endpoints)
 - updated documentation/readme
 - a new minor package version (0.1.7)
 
Please review according to https://github.com/CODAIT/node-red-contrib-model-asset-exchange/wiki/New-node-validation-checklist

### CODE REVIEW

File: inception-resnet-v2/node.html [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+        <dt>payload <span class="property-type">object</span></dt>
+        <dd>Returns the metadata associated with the model.</dd>
+    </dl>
+
+</script>
+<script type="text/javascript">
+    RED.nodes.registerType('inception-resnet-v2-service', {
+        category: 'config',
+        defaults: {
+            host: { value: "http://max-inception-resnet-v2.max.us-south.containers.appdomain.cloud", required: true },
```
Comment (zuhito) [inline inception-resnet-v2/node.html]: @ptitzler In terms of data encryption, I think that `https://max-inception-resnet-v2.max.us-south.containers.appdomain.cloud` is better than the URL which starts with `http://`.

**Comment by zuhito** [comment]
Thank you for the fix. No problem. 👍

File: inception-resnet-v2/node.html (hunk 1 above)
Comment (ptitzler) [reply inception-resnet-v2/node.html]: I changed all `http` references to `https`.

**Comment by ptitzler** [comment]
Some of the tests (even for nodes that have not been modified) sometimes "randomly" fail when run by Travis-CI. I have not observed the same behavior when I ran those tests on my machine.

**Pull request merged**